src/tool.js

```diff
@@ -0,0 +1,15 @@
+module.exports.formGenerator = (inputs=[]) => {
+  let html = '<form>'
+  for (let i = 0; i < inputs.length; i++) {
+    if (
+      inputs[i].type === 'text' ||
+      inputs[i].type === 'email' ||
+      inputs[i].type === 'number' ||
+      inputs[i].type === 'submit'
+    ) {
+      html += `<input type="${inputs[i].type}"></input>`
```

Comment:
Your logic is generating input tags that are unusable. There is no id or name attribute attached that would allow capturing of this form data. Additionally, labels, placeholder text, and any common elements found within forms is not accommodated here, and while not specifically asked for, it is low hanging fruit to accommodate for.